packages/react-cosmos/src/config/shared.ts

```diff
@@ -12,6 +12,13 @@ export type CosmosConfig = {
   // unspecified IPv4 address (0.0.0.0) otherwise.
   hostname: null | string;
   port: number;
+  https?: boolean;
+  httpsOptions?: {
+    keyPath?: string;
+    certPath?: string;
+  };
+  httpsKeyPath?: string;
+  httpsCertPath?: string;
```

Comment:
I don't think duplicating the types like this is needed. See my comment at `createCosmosConfig.ts:16`